docs/features.rst

```diff
@@ -72,12 +72,7 @@ Starting from Python 3.7 an even better approach is using `importlib.resources`_
 Note that we need a proper package structure in this case, i.e. directories need
 to contain ``__init__.py`` and we only specify the file ``data.txt``, no path is allowed.
 The library importlib_resources_ provides a backport of this feature.
-Even another way, provided by `setuptools`_'s  `pkg_resources`_ is::
 
-    from pkg_resources import resource_string
-    data = resource_string(__name__, 'path/to/my/data/relative/to/module.txt')
-
-Yes, actually "there should be one-- and preferably only one --obvious way to do it." ;-)
```

Comment:
Sorry for removing the funny comment, I feel like since recommending just an old way and a new way of doing that, instead of 3 ways, it looses a little bit its impact.

Reply:
Since it wasn't so funny, I guess it's okay to remove it ;-)